lib/titleConvertor.js

```diff
@@ -0,0 +1,17 @@
+const titleConvertor = (name) => {
```

Comment:
Could this name be made clearer to do you think? What is this function doing? Is it converting something to or from a title?